quarkus-spring-aop-api/pom.xml

```diff
@@ -73,6 +73,7 @@
                                         <include>org/springframework/aop/support/AbstractPointcutAdvisor**</include>
                                         <include>org/springframework/aop/target/EmptyTargetSource**</include>
                                         <include>org/springframework/aop/target/SingletonTargetSource**</include>
+                                        <include>org/springframework/aot/hint/annotation/Reflective**</include>
```

Comment:
If they are all annotations, I think I would add everything in this package. It doesn't hurt.